public_html/wp-content/plugins/pattern-directory/includes/pattern-post-type.php

```diff
@@ -298,7 +298,7 @@ function register_rest_fields() {
 		'pattern_content',
 		array(
 			'get_callback' => function() {
-				return wp_kses_post( get_the_content() );
+				return decode_pattern_content( get_the_content() );
```

Comment:
Do you think it's safe not to sanitize through `wp_kses_post` here? Check my comment: https://github.com/WordPress/gutenberg/issues/47636#issuecomment-1423783368

> The issue here is a result of GB sanitizing the pattern content through `wp_kses_post`. The flow there is:
- `wp_kses_normalize_entities` replaces `&` to `&amp;`
- `wp_pre_kses_block_attributes` replaces `&` to `\u0026` that results in `\u0026amp;` in the final output.

>The above though is for security reasons and normally is not an issue for block attributes that are not URLs. In the `Cover`
block case though it is a problem, as it results to a malformed URL.